src/Backend/One/OneToolchain.ts

```diff
@@ -14,33 +14,188 @@
  * limitations under the License.
  */
 
+import {execSync} from 'child_process';
+import * as vscode from 'vscode';
+
 import {Backend} from '../Backend';
 import {Command} from '../Command';
 import {Compiler} from '../Compiler';
 import {Executor} from '../Executor';
-import {Toolchains} from '../Toolchain';
+import {PackageInfo, ToolchainInfo, Toolchains} from '../Toolchain';
+import {DebianArch, DebianRepo, DebianToolchain} from '../ToolchainImpl/DebianToolchain';
+import {Version} from '../Version';
+
+class OneDebianToolchain extends DebianToolchain {
+  run(cfg: string): Command {
+    this.prepare();
+    let cmd = new Command('onecc-docker');
+    cmd.push('-C');
+    cmd.push(cfg);
+    return cmd;
+  }
+}
 
 class OneCompiler implements Compiler {
   private readonly toolchainTypes: string[];
+  private readonly toolchainName: string;
+  private readonly debianRepo: DebianRepo;
+  private readonly debianArch: DebianArch;
 
   constructor() {
-    this.toolchainTypes = [];
+    this.toolchainName = 'onecc-docker';
+    this.toolchainTypes = ['latest'];
```

Comment:
Q. Is it always latest? What if docker registry is not updated yet?

Reply:
Yes.